fuel-core/src/tx_pool.rs

```diff
@@ -37,6 +37,8 @@ pub enum Error {
     Database(Box<dyn StdError>),
     #[error("unexpected block execution error {0:?}")]
     Execution(crate::executor::Error),
+    #[error("Tx is invalid, insertion failed {0:?}")]
+    Other(#[from] anyhow::Result),
```

Comment:
whoops meant to use this

```suggestion
    Other(#[from] anyhow::Error),
```